test/libsolidity/semanticTests/array/copying/string_calldata_to_bytes_calldata.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+    function f(string calldata s) public returns (bytes calldata m) {
+        return bytes(s);
```

Comment:
Should also have tests for the other direction, bytes -> string.

Reply:
Added